scripts/bootstrap/buildbootstrapcli.sh

```diff
@@ -288,9 +288,14 @@ sed \
     -e 's/runtime\.linux-x64/runtime.'$__runtime_id'/g' \
     -e 's/runtimes\/linux-x64/runtimes\/'$__runtime_id'/g' \
     -e 's/Version=v\([0-9].[0-9]\)\/linux-x64/Version=v\1\/'$__runtime_id'/g' \
-    -e 's/"runtimes": {/&\n    "'$__runtime_id'": [\n      "unix", "unix-x64", "any", "base"\n    ],/g' \
 $__seedclipath/shared/Microsoft.NETCore.App/$__frameworkversion/Microsoft.NETCore.App.deps.json \
 >$__frameworkpath/Microsoft.NETCore.App.deps.json
 
+# add the new RID to the list of runtimes iff it does not already exist (sed inplace)
+grep -q "\"$__runtime_id\"" $__frameworkpath/Microsoft.NETCore.App.deps.json || \
+sed -i \
+    -e 's/"runtimes": {/&\n    "'$__runtime_id'": [\n      "unix", "unix-'$__build_arch'", "any", "base"\n    ],/g' \
+$__frameworkpath/Microsoft.NETCore.App.deps.json
```

Comment:
`runtimes` already contains `"linux-arm64" : { ... }`

Reply:
A nit - could you please modify the grep condition to include the `: [` too to be a little more resilient to a case when the Microsoft.NETCore.App.deps.json would get the `\"$__runtime_id\"` added to some other place for a different purpose?

Reply:
Done